portable/ThirdParty/GCC/Xtensa_ESP32/port.c

```diff
@@ -56,12 +56,14 @@
  *  https://www.FreeRTOS.org/labs - Where new FreeRTOS products go to incubate.
  *  Come and try FreeRTOS+TCP, our new open source TCP/IP stack for FreeRTOS.
  *
- *  http://www.OpenRTOS.com - Real Time Engineers ltd. license FreeRTOS to High
+ *  https://www.highintegritysystems.com/openrtos/ - Real Time Engineers ltd.

```

Comment:
Please reformat so the lines break at 80 characters - and in the next change block below.

Reply:
it is at 80 chars, or you want some modifications in style?